scripts/contributing.js

```diff
@@ -0,0 +1,36 @@
+/* global jQuery*/
+
+(function($){
+    var githubIssuesEndpoint = 'https://api.github.com/repos/babel/babel/issues';
```

Comment:
I get a 404 for that endpoint even when using my API key :thinking:. Same with my other repositories. May just be broken.